Branch-SDK/src/main/java/io/branch/referral/validators/LinkingValidatorConstants.java

```diff
@@ -0,0 +1,35 @@
+package io.branch.referral.validators;
+
+public class LinkingValidatorConstants {
+        public static final String canonicalURLPromptText = "Please paste in a web link for the $canonical_url";
+        public static final String deeplinkPathPromptText = "Please paste in a value for the $deeplink_path";
+        public static final String customKeyPromptText = "Please enter your custom key and value for routing";
+        public static final String step1ButtonText = "Next";
+        public static final String step2ButtonText = "  Generate Links for Testing  ";
+        public static final String step3ButtonText = "Done";
+        public static final String canonicalUrlKey = "$canonical_url";
+        public static final String deeplinkPathKey = "$deeplink_path";
+
+        public static final String linkingValidatorRow1Title = "Link using App Link";
+        public static final String linkingValidatorRow2Title = "Link using URI scheme";
+        public static final String linkingValidatorRow3Title = "Web-only link";
+        public static final String linkingValidatorRow4Title = "Link with missing data";
+        public static final String linkingValidatorRow5Title = "Warm start use case";
+        public static final String linkingValidatorRow6Title = "Foreground click use case";
+
+        public static final String infoButton1Copy = "Verifies that Universal Links / App Links are working correctly for your Branch domain";
```

Comment:
Nit: should these variables have `Copy` in them?

Reply:
Copy is marketing lingo for "text" but I realize it could be confusing so I am renaming these variables to say "Text" in the name instead of "Copy". Good point!